### PR TITLE
Update cloud backup articles

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-17f056a7d6/how-to/).

**Comment by catlook** [comment]
Ben Meyer approved the changes!

File: content/cloud-backup/update-the-rackspace-cloud-backup-agent.md [hunk 1]
```diff
@@ -98,9 +120,14 @@ Example init/startup scripts are included in the tarball.
...
+privileges. You might want to review
+[Install the Cloud Backup agent on Windows](/how-to/rackspace-cloud-backup-install-the-agent-on-windows)
+before performing these steps.
 
-Because the Windows agent includes an automated updater, you should rarely need to update the Windows agent manually. This section provides different options to use if you need to manually update.
+Because the Windows agent includes an automated updater, you should rarely need
+to update the Windows agent manually. This section provides different options
+to use if you need to manually update.
 
 #### Recommended method
```
Comment (nimbinatus) [inline content/cloud-backup/update-the-rackspace-cloud-backup-agent.md]: The bullets under number 1 need to be indented a few spaces more to get the numbering working.

File: content/cloud-backup/rackspace-cloud-backup-install-the-agent-on-linux.md [hunk 2]
```diff
@@ -5,101 +5,133 @@ title: Install or update the Cloud Backup agent on Linux
...
     per the preceding step.
 
-7.  Run the agent with the `--configure` option to configure it.
+10.  Run the agent with the `--configure` option to configure it.
 
         sudo /usr/local/bin/driveclient --configure --username <username> --apikey <apiKey> --flavor <flavor> --datacenter <dataCenter> --apihost api.drivesrvr.com
 
-    - Use your Rackspace Cloud account username and API key for `<username>` and `<apiKey>`. For information about how to find your API key, see [View and reset your API key.](/how-to/view-and-reset-your-api-key)
-    - The value for `<flavor>` is `privatecloud`, `raxcloudserver`, or `dedicated`. Typically it is `raxcloudserver`.
+    - Use your Rackspace Cloud account username and API key for `<username>`
```
Comment (nimbinatus) [inline content/cloud-backup/rackspace-cloud-backup-install-the-agent-on-linux.md]: this line down to 11 seems to be accidentally appearing as a code block?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-43e62c53a5/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b155d77911/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9e4c2ec31e/how-to/)

**Pull request merged**